app/1.3.x/db-less-admin-api.md

```diff
@@ -1457,15 +1457,15 @@ HTTP 200 OK
 
 ---
 
-### Retrieve Ca_certificate
+### Retrieve Ca Certificate
 
-##### Retrieve Ca_certificate
+##### Retrieve CA Certificate
```

Comment:
(it's odd that this one didn't get auto-capitalized, looks like a tiny problem in the script)

Reply:
https://github.com/Kong/docs.konghq.com/pull/1421/commits/1473b97b6f217c92df2729e97bcdfcf964f82441 seems to fixed it :)